x-pack/plugins/security_solution/public/detections/components/rules/description_step/translations.tsx

```diff
@@ -96,3 +96,24 @@ export const THRESHOLD_RESULTS_AGGREGATED_BY = i18n.translate(
     defaultMessage: 'Results aggregated by',
   }
 );
+// TODO: maybe `depricated` instead? Since its tied to mitre currently
+export const UNKNOWN_TACTIC = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleDescription.unknownTactic',
+  {
+    defaultMessage: 'Unknown Tactic',
```

Comment:
Looking for opinions on this `Unknown ...` label. Given the use case would it be better to put `Deprecated ...` or some combo? Something totally different?

Reply:
went with the direct `technique.name` field